compiler/rustc_passes/src/check_attr.rs

```diff
@@ -2214,12 +2331,15 @@ fn check_non_exported_macro_for_invalid_attrs(tcx: TyCtxt<'_>, item: &Item<'_>)
 }
 
 fn check_mod_attrs(tcx: TyCtxt<'_>, module_def_id: LocalDefId) {
-    let check_attr_visitor = &mut CheckAttrVisitor { tcx };
+    let check_attr_visitor = &mut CheckAttrVisitor { tcx, abort: Cell::new(false) };
     tcx.hir().visit_item_likes_in_module(module_def_id, check_attr_visitor);
     if module_def_id.is_top_level_module() {
         check_attr_visitor.check_attributes(CRATE_HIR_ID, DUMMY_SP, Target::Mod, None);
         check_invalid_crate_level_attr(tcx, tcx.hir().krate_attrs());
     }
+    if check_attr_visitor.abort.get() {
+        tcx.sess.abort_if_errors()
+    }
```

Comment:
This is to suppress the errors like in https://github.com/rust-lang/rust/issues/102923. If omitted the compiler will print both those and these new errors.

If this is not OK or there's a better way to suppress them, please let me know.

Reply:
This isn't great, yeah :/

Reply:
If this is what it takes, I guess we'll have to live with it until we find an alternative.